templates/express/files/index.js

```diff
@@ -0,0 +1,115 @@
+var express = require('express');
+const mysql = require('mysql2/promise');
+const config = require("platformsh-config").config();
+
+// Define the application.
+var app = express();
+
+async function mariadbTest() {
+    try {
+
+        var results = [];
+
+        // Connect to MariaDB.
+        const credentials = config.credentials('database');
+        const connection = await mysql.createConnection({
+            host: credentials.host,
+            port: credentials.port,
+            user: credentials.username,
+            password: credentials.password,
+            database: credentials.path,
+        });
+
+        results[0] = `Connected to MariaDB at ${credentials.host}.`
+
+        // Add the table.
+        tableName = "userinfo";
+
+        let sql = '';
+
+        // Create a table.
+        sql = `CREATE TABLE IF NOT EXISTS ${tableName} (
+          uid INT(10) NOT NULL AUTO_INCREMENT,
+          username VARCHAR(64) NULL DEFAULT NULL,
+          departname VARCHAR(128) NULL DEFAULT NULL,
+          created DATE NULL DEFAULT NULL,
+          PRIMARY KEY (uid)
+          ) DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;`;
+
+        await connection.query(sql);
+
+        results[1] = `Created table "${tableName}".`;
+
+        // Insert data.
+        values = `(username, departname, created)`;
+        newRow = `('platform', 'Deploy Friday', '2019-06-17')`;
+
+        sql = `INSERT INTO ${tableName} ${values} VALUES ${newRow};`
```

Comment:
This should be using a prepared statement.  Literal values belong in a query precisely never. :smile:  (Unless it's truly a single literal string for the whole query.  Although even that is debatable.)